sources/osg/Matrix.js

```diff
@@ -1224,6 +1224,28 @@ define( [
             };
         } )(),
 
+        preMultScale: function ( x, y, z, m ) {
```

Comment:
Check how it's done for preMultTranslate, the order argument is not the same, matrix argument is always first.
The other point should be maybe to change premulttranslate to use it without array but scalar instead.
Or maybe we can do it in another commit. But for consistent both method should work the same !fix